test/controllers/returns/StartYourReturnControllerSpec.scala

```diff
@@ -137,8 +138,8 @@ class StartYourReturnControllerSpec extends SpecBase with MockitoSugar  {
         val result = route(application, request).value
 
         status(result) mustEqual SEE_OTHER
-        redirectLocation(result).value mustEqual controllers.returns.credits.routes.WhatDoYouWantToDoController.onPageLoad(NormalMode).toString
-
+        assert(redirectLocation(result).value == controllers.returns.credits.routes.WhatDoYouWantToDoController.onPageLoad(NormalMode).toString ||
```

Comment:
hmmm not sure about this, a test should assert it is deffo one or the other. 
If app config is mocked you can force the toggle one way or the or the other so the assertion can be deterministic.

Reply:
Good point, i've changed this to use the mock config instead